rules-reviewed/camel3/camel2/java-generic-information.windup.xml

```diff
@@ -110,7 +110,7 @@
                                 href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_generic_information"/>
                         <quickfix type="REPLACE" name="Registry-replace">
                             <replacement>org.apache.camel.support.DefaultRegistry</replacement>
-                            <search>org.apache.camel.impl.{registry}</search>
+                            <search>org.apache.camel.impl.{*}</search>
```

Comment:
I tested this wildcard `{*}` search pattern and it doesn't work.
The solution i found (since the `registry` parameter has just 2 values) is to create two different quickfixes, one for each value.
```suggestion
                            <search>org.apache.camel.impl.PropertyPlaceholderDelegateRegistry</search>
                        </quickfix>
                        <quickfix type="REPLACE" name="CompositeRegistry-replace">
                            <replacement>org.apache.camel.support.DefaultRegistry</replacement>
                            <search>org.apache.camel.impl.CompositeRegistry</search>
```

In this way, in the IDE plugin, the line of code that triggers the rule will have 2 options to choose the quickfix to be applied (see screenshot above) which seems to me a nice way to manage this case.
![Screenshot from 2020-03-06 11-23-22](https://user-images.githubusercontent.com/7288588/76075655-ac184f80-5f9d-11ea-9dba-b5d3cca4b20d.png)